test/mocha/cli.js

```diff
@@ -55,15 +55,15 @@ describe("bin/uglifyjs", function () {
         });
     });
     it("Should append source map to output when using --source-map url=inline", function (done) {
-       var command = uglifyjscmd + " test/input/issue-1323/sample.js --source-map url=inline";
+        var command = uglifyjscmd + " test/input/issue-1323/sample.js --source-map url=inline";
```

Comment:
Also fixed missing paddings throughout this file.

Please use https://github.com/mishoo/UglifyJS2/pull/1941/files?w=1 to view the actual code changes.